content/en/agent/cluster_agent/clusterchecks.md

```diff
@@ -177,15 +255,15 @@ spec:
         run: my-nginx
 ```
 
-In addition, each pod should be monitored with the [NGINX check][10], as it enables the monitoring of each worker as well as the aggregated service.
+In addition, each pod should be monitored with the [NGINX check][9], as it enables the monitoring of each worker as well as the aggregated service.
 
 ## Troubleshooting
 
 Due to the distributed nature of cluster checks, troubleshooting them is a bit more involved. The following sections explain the dispatching process and the associated troubleshooting commands.
 
 ### Kubernetes: find the leader Cluster Agent
 
-When leader election is enabled, only the leader serves Cluster Check configurations to the node-based Agents. The name of the leader is available in the `datadog-leader-election` ConfigMap:
+When leader election is enabled, only the leader serves Cluster Check configurations to the node-based Agents. If you only have 1 replica of the Cluster Agent pod running, that will be the leader. Otherwise, you can identify the name of the leader in the `datadog-leader-election` `ConfigMap`:
```

Comment:
```suggestion
When leader election is enabled, only the leader serves Cluster Check configurations to the node-based Agents. If only one replica of the Cluster Agent pod is running, it is the leader. Otherwise, you can identify the name of the leader in the `datadog-leader-election` `ConfigMap`:
```